README.md

```diff
@@ -1,6 +1,126 @@
-# Nyuntam (Kompress)
-Kompress is NyunAI's robust compression module that effortlessly compresses and accelerates a diverse range of deep learning tasks and models. This module incorporates state-of-the-art compression methods, distributed between [nyuntam-text-generation](https://github.com/nyunAI/nyuntam-text-generation) and [nyuntam-vision](https://github.com/nyunAI/nyuntam-vision), implementing the efficiencies for text-generation (large language models) and vision tasks respectively.
+# Nyuntam
 
-For installations and usage checkout -
-- [nyuntam-text-generation](https://github.com/nyunAI/nyuntam-text-generation/blob/main/README.md)
-- [nyuntam-vision](https://github.com/nyunAI/nyuntam-vision/blob/main/README.md)
+**Nyuntam** is NyunAI's advanced suite designed to optimize, adapt, and accelerate a wide array of deep learning models across various domains. The repository is structured into several submodules, each targeting specific tasks:
+
+- **Nyuntam Text Generation**: Focuses on compressing large language models for text generation tasks.
+- **Nyuntam Vision**: Tailored for compressing and optimizing vision models.
+- **Nyuntam Adapt**: A robust module for fine-tuning and transfer learning across both vision and language models.
```

Comment:
lets also mention a few exampler algos for adapt something like - A robust module for fine-tuning and transfer learning across both vision and language models leveraging SoTA PEFT, full-finetuning and GPU parallelism.